cypress/TestCases/FireboltCertification/LifecycleTestcases/LifecycleBackground.feature

```diff
@@ -35,26 +35,26 @@ Feature: Lifecycle_Background
         Given the environment has been set up for 'lifecycle' tests
         And 3rd party 'certification' app is launched with 'background' state
         When '3rd party app' transitions to state 'background'
-        Then '3rd party app' is in 'background' state
+        Then '3rd party app' will stay in 'background' state
         
     @Lifecycle @coreSDK @notSupported
     Scenario: Lifecycle 2.4.3 Cannot Background app in Unloading state
         Given the environment has been set up for 'lifecycle' tests
         And 3rd party 'certification' app is launched with 'unloading' state
         When '3rd party app' transitions to state 'background'
         And AppObject state for '3rd party App' is set to 'unloading'
-        Then '3rd party app' is in 'unloading' state
+        Then '3rd party app' will be in 'unloading' state
```

Comment:
Wouldn't this be a `stay in` scenario since the app isn't actually transitioning?

Reply:
updated